apl-utils/src/test/java/com/apollocurrency/aplwallet/apl/util/cache/InMemoryCacheManagerTest.java

```diff
@@ -125,6 +126,29 @@ void testCacheEvictions() {
         assertArrayEquals(Long.toUnsignedString(i*1000L).getBytes(), cache.getIfPresent(i));
     }
 
+    @Test
+    void testMemCalculator(){
+        int size = new InMemoryCacheManager.MemoryUsageCalculator(64)
+                .startObject()//+16
+                .addBoolean() //+1
+                .addByte()//+1
+                .addChar()//+2
+                .addInt() //+4 + padding=2
```

Comment:
Looks like padding here is not required and should be applied after all added fields

Reply:
sure